packages/core/src/mappings/BottomSheet.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  COMPONENT_TYPES,
+  createStaticBoolProp,
+  CONTAINER_COMPONENT_STYLES_SECTIONS,
+  Triggers,
+  createActionProp,
+  createColorProp,
+  createStaticNumberProp,
+  createArrayProp,
+} from "@draftbit/types";
+
+export const SEED_DATA = {
+  name: "Bottom Sheet",
+  tag: "BottomSheet",
+  description:
+    "A draggable Bottom sheet that snaps to specific points. Renders children in ScrollView",
+  category: COMPONENT_TYPES.bottomsheet,
+  stylesPanelSections: CONTAINER_COMPONENT_STYLES_SECTIONS,
+  layout: { paddingHorizontal: 16, paddingVertical: 10 },
```

Comment:
Does it make sense to have these defaults here or just in the component itself?

Reply:
I felt like having them here clearly shows the user the default applied padding. But then again, "auto" is a good representation of the default padding. I'll remove it from the mapping.